tICA/scripts/feature_helpers/ComputeDVARSandGS.m

```diff
@@ -0,0 +1,131 @@
+function ComputeDVARSandGS(StudyFolder,Subjlist, hp, MRFixConcatName, fMRINames, RegString, ProcString, RecleanMode)
+%UNTITLED Summary of this function goes here
+%   Detailed explanation goes here
```

Comment:
No help/usage statement.

Reply:
This is an internal helper to internal matlab code to an internal shell script.  Resolving as not user-facing.